src/server.js

```diff
@@ -12,9 +12,13 @@ import * as dotenv from 'dotenv';
 import { FindColors } from './findColors.js';
 import { getPaletteTitle } from './generatePaletteName.js';
 import { svgTemplate } from './colorSwatchSVG.js';
+import { createColorRecord } from './lib.js';
+import { initDatabase, addResponseToTable } from './database.js';
 
 dotenv.config();
 
+initDatabase(process.env.SUPRABASEURL, process.env.SUPRABASEKEY);
```

Comment:
what happens if the server is not available? Would be cool if the API still works, but just does not log anything to the DB.

Reply:
This would fail on app start. But I did add error catching for the actual save call. So if it fails, the sever will still continue to work.